source/agora/flash/OnionPacket.d

```diff
@@ -200,12 +200,11 @@ public OnionPacket createOnionPacket (in Hash payment_hash,
 
         packet.encrypted_payloads[last_index] = encrypted_payload;
         last_index--;
-        if (!forward_amount.sub(hop.fee))
+        if (!forward_amount.add(hop.fee))
```

Comment:
I think we have `mustAdd/mustSub` for use cases like this.

Reply:
https://github.com/bpfkorea/agora/blob/df511f4cbfaf6cca04b92fbe1be5175eac85911f/source/agora/common/Amount.d#L301